packages/core/lib/Configuration.ts

```diff
@@ -20,15 +20,22 @@ import Yargs = require("yargs");
 import * as path from "path";
 import shell = require("shelljs");
 
-export let CONFIG: Readonly<ArgumentsObject>;
+export type RemoveIndex<T> = {
```

Comment:
Can you add a comment explaining the purpose of the type

Reply:
Fixed.